tutornotes/plugin.py

```diff
@@ -20,6 +20,8 @@
         "HOST": "notes.{{ LMS_HOST }}",
         "MYSQL_DATABASE": "notes",
         "MYSQL_USERNAME": "notes",
+        "CODE_REPO": "https://github.com/edx/edx-notes-api",
```

Comment:
The value should be: "https://github.com/openedx/edx-notes-api"

Reply:
thanks @regisb. Already addressed